app/models/pipeline_run.rb

```diff
@@ -58,6 +66,10 @@ def self.in_progress
       .where(finalized: 0)
   end
 
+  def self.results_in_progress
+    where(results_finalized: 0)
```

Comment:
Yeah this seems not to be the case if alignment loader and postprocessing loader are executed out of order by resque, and the postprocessing loader fails.

Reply:
However, in that case, there is still no need to run the result monitor again, which is the point of the results_in_progress method. Alignment loader is already enqueued.

Reply:
Got it.   Just slightly confused by the name `results_finalized`.